module/dkg.go

```diff
@@ -61,15 +61,18 @@ type DKGController interface {
 	// Shutdown stops the controller regardless of the current state.
 	Shutdown()
 
-	// GetArtifacts returns the private and public shares, as well as the set of
-	// public keys computed by DKG.
-	GetArtifacts() (crypto.PrivateKey, crypto.PublicKey, []crypto.PublicKey)
-
 	// Poll instructs the controller to actively fetch broadcast messages (ex.
 	// read from DKG smart contract). The method does not return until all
 	// received messages are processed.
 	Poll(blockReference flow.Identifier) error
 
+	// GetArtifacts returns the private and public shares, as well as the set of
+	// public keys computed by DKG.
```

Comment:
```suggestion
	// GetArtifacts returns our node's private key share, the group public key, 
	// and the list of all nodes' public keys (including ours), as computed by the DKG.
```

I might have the terminology wrong here, but I think this returns:
1. my private key share
2. the shared group public key
3. the public key shares for all participants, including my public key